roboteam_ai/src/utilities/OffensiveCoach.cpp

```diff
@@ -0,0 +1,317 @@
+//
```

Comment:
I like the use of exponential functions to model probabilities

Reply:
Thanks!

Reply:
Thanks!